doc/onboarding.md

```diff
@@ -64,31 +64,29 @@ onboarding session.
     * will also come more naturally over time
 
 
-  * reviewing:
-    * primary goal is for the codebase to improve
-    * secondary (but not far off) is for the person submitting code to succeed
-      * helps grow the community
-      * and draws new people into the project
```

Comment:
I was trying to decide which line to remove because they seemed to be saying the same thing just using different words.

Then I thought the document is probably fine without either of them and removed them both.

I think what those two lines say are implied by the line above them.

But I don't feel that strongly about it and can restore one or the other if anyone feels strongly about it.